onebox/start_onebox.sh

```diff
@@ -88,6 +87,32 @@ test -d recycle_bin3 && rm -rf recycle_bin3
                    --zk_cluster=${ZK_CLUSTER}\
                    --zk_root_path=/onebox > ns3.log 2>&1 &
 
+# start TaskManager if exists
+if [ -d "../java/openmldb-taskmanager/target/" ]
+then
+  # Change to binary directory
+  pushd ../java/openmldb-taskmanager/target/openmldb-taskmanager-binary/bin > /dev/null
```

Comment:
relative path not always work when executing script in different directory. instead, simply add
```sh
cd "$(dirname "$0")/.."
```
in line 18
and assume all path below is relative to the project  root